### PR TITLE
Upgraded requirements

### LINKED ISSUE #20: Use Django 1.9

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #20

### CODE REVIEW

File: {{ cookiecutter.project_name }}/test_requirements.txt [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-pytest-pep8
-pytest-pylint
-pytest-cov
-pytest-django
-pytest-watch
-mock
-semantic_version
+factory_boy==2.6.0
+fake-factory==0.5.3
+ipython==4.1.2
```
Comment (giocalitri) [inline {{ cookiecutter.project_name }}/test_requirements.txt]: I would leave `ipython` without a version

**Comment by giocalitri** [comment]
Just a small change, otherwise :+1:

File: {{ cookiecutter.project_name }}/test_requirements.txt (hunk 1 above)
Comment (noisecapella) [inline {{ cookiecutter.project_name }}/test_requirements.txt]: I'm fine doing it, but why?

Comment (giocalitri) [inline {{ cookiecutter.project_name }}/test_requirements.txt]: it is not worth to lock this on a specific version

**Pull request merged**